site/docs/samples/notebook.md

```diff
@@ -1,162 +1,239 @@
-1. Create a Mesh for Data runtime environment for the notebook
-1. Read the dataset and observe policies applied seamlessly
+This sample uses the [Synthetic Financial Datasets For Fraud Detection](https://www.kaggle.com/ntnu-testimon/paysim1/data) dataset[^1] as the data that the notebook needs to read. Download and extract the file to your machine. You should now see a file named `PS_20174392719_1491204439457_log.csv`. Alternatively, you may download [`data.csv`](https://github.com/ibm/the-mesh-for-data/blob/master/samples/kubeflow/data.csv) which includes just the first 100 lines of this dataset.
 
-## Getting started
+[^1]: Created by NTNU and shared under the ***CC BY-SA 4.0*** license.
 
-Completed all the steps in [quick start guide](../get-started/quickstart.md).
+Upload the CSV file to an object storage of your choice such as AWS S3, IBM Cloud Object Storage or Ceph.
+Make a note of the service endpoint, bucket name, and access credentials. You will need them later.
 
-## Prepare the dataset for the sample notebook
+??? tip "Setup and upload to MinIO"
 
-1. Upload [data.csv](https://github.com/ibm/the-mesh-for-data/blob/master/samples/kubeflow/data.csv) to an object storage of your choice
-    `data.csv` contains the first 100 rows from the following [data set](https://www.kaggle.com/ntnu-testimon/paysim1/data) created by NTNU, and it is shared under the ***CC BY-SA 4.0*** license.
-1. Update ```samples/kubeflow/example_transactions.csv.json``` with the location of the dataset, The location is encoded in the `fullPath` field as follows:
-    - Bucket: Change the `bucket` value from `m4d-bucket-example` to the bucket name where the dataset reside.
-    - endpoint: Change the S3 endpoint name from `s3.eu-de.cloud-object-storage.appdomain.cloud` to the object storage endpoint
-    - object_key: If needed, change object_key from `data.csv` to the object name that you used in the previous step.
-    For more information on the content please see the comments in [third_pary/egeria/usage/create_new_asset.sh](https://github.com/ibm/the-mesh-for-data/blob/master//third_party/egeria/usage/create_new_asset.sh) for more details.
-1. Register the dataset in the catalog
-
-    - Setup port forwarding for communicating with Egeria
-
-        ```bash
-        cd third_party/egeria/usage
-        kubectl port-forward -n egeria-catalog svc/lab-core 9443:9443 &
-        ```
-    - Wait for the port-forward to take effect.
-
-    - Register the dataset in the catalog with the tag 'finance'.
+    For experimentation you can install MinIO to your cluster instead of using a cloud service.
     
-        ```bash
-        ./create_new_asset.sh ../../../samples/kubeflow/example_transactions.csv.json 'finance'
-        ```
-    - Cleanup the port forwarding using the following.
+    1. Install Minio to the currently active namespace:
+      ```bash
+      helm repo add minio https://helm.min.io
+      helm install --wait minio minio/minio
```

Comment:
This step gives "Error: timed out waiting for the condition". If I start the command again it says "Error: cannot re-use a name that is still in use" . can we provide a fix for this in the documentation? I am testing this on kind. The only way to retry seems to be I have to reinstall the kind cluster and try the command again.